attribute-service-api/src/main/proto/org/hypertrace/core/attribute/service/v1/attribute_metadata.proto

```diff
@@ -28,6 +28,7 @@ enum AttributeScope {
   CLUSTERS_SNAPSHOT = 17;
   DOMAIN = 18;
   ACTOR = 19;
+  SPAN_SENSITIVE_DATA = 20;
```

Comment:
Can we rename it to `SENSITIVE_SPAN` or `SENSITIVE_EVENT`? We interchangeably use `SPAN` and `EVENT` in attribute scopes

Reply:
No EVENT please. EVENT was wrongly named. It should have been SPAN. So lets not continue to name new enums that way.